build/dependencies.yaml

```diff
@@ -1,7 +1,7 @@
 dependencies:
   # agnhost: bump this one first
   - name: "agnhost"
-    version: "2.27"
+    version: "2.28"
```

Comment:
Don't change this just yet (since this version does not exist in the gcr repository)

Reply:
oh.. the dependencies job is failing with a `test/images/agnhost/VERSION` mismatch.

Reply:
@dims I think that is correct
>   # agnhost: bump this one first
there is another place that is the one that has to be bumped later, once the promotion has been done